tools/packaging/windows/aws-otel-collector.wxs

```diff
@@ -20,6 +20,12 @@ SummaryCodepage='1252'
 InstallScope="perMachine"
 Platform="x64"
     />
+
+    <MajorUpgrade
+    AllowDowngrades="no" DowngradeErrorMessage="A later version of ADOT Collectot is already installed. Setup will now exit"
+    AllowSameVersionUpgrades="yes"
```

Comment:
Do we have to set this to no. As we don't usually look after the fourth version field and may not be useful setting.

'''
This is useful when two product versions differ only in the fourth version field. MSI specifically ignores that field when comparing product versions, so two products that differ only in the fourth version field are the same product and need this attribute set to yes to be detected.
'''